Geometry/TrackerGeometryBuilder/interface/phase1PixelTopology.h

```diff
@@ -29,6 +30,66 @@ namespace phase1PixelTopology {
                                           };
 
 
+  template<class Function, std::size_t... Indices>
+  constexpr auto make_array_helper(Function f, std::index_sequence<Indices...>)
+  -> std::array<typename std::result_of<Function(std::size_t)>::type, sizeof...(Indices)>
+  {
+    return {{ f(Indices)... }};
+  }
+
+  template<int N, class Function>
+  constexpr auto make_array(Function f)
```

Comment:
if I understand correctly the code, this function returns an std::array with N elements, that are the results of calling `f(i)` for `i` in `0 .. (N-1)`.
If that is correct, I would suggest to use a different name, like `map_to_array` or `apply_to_array` or `array_apply` or ...

Reply:
copied from internet...
so feel free to change name

Reply:
there is something similar in future std called make_array
https://en.cppreference.com/w/cpp/experimental/make_array

Reply:
Yes, that's why I would like to avoid the `make_array` name here, since the functionality is different.